src/components/celestialmap/add_custom_data.js

```diff
@@ -0,0 +1,259 @@
+import { default_custom_object_styles } from './skymap_config'
```

Comment:
Clear to me. 
My only question is if we are displaying a chart legend or not on the targets page along with the skychart, especially since not all object names will be shown on the skychart due to overlapping, and students may not have learned yet or memorized the different types of objects they can point at (depending on where they are at with lessons). (I do not see a legend with the current dev version). 
If this could be distracting or take up too much space, maybe a toggle on/off legend of symbols with brief definitions?
This might tie into the education aspects of tuning the user interface. 
This is just an idea.

Reply:
A legend is a great idea. Implementation will be a little tricky, both positioning it in the layout and rendering the same symbols (since they're html canvas drawings, not something easy like jpgs). But definitely doable.